src/main/java/frc/trigon/robot/subsystems/arm/toohardarm/ToohardArmIO.java

```diff
@@ -0,0 +1,86 @@
+package frc.trigon.robot.subsystems.arm.toohardarm;
+
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix6.StatusSignal;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.math.util.Units;
+import frc.trigon.robot.subsystems.arm.ArmIO;
+import frc.trigon.robot.subsystems.arm.ArmInputsAutoLogged;
+import frc.trigon.robot.utilities.Conversions;
+
+public class ToohardArmIO extends ArmIO {
+    private final CANSparkMax
+            masterAngleMotor = ToohardArmConstants.MASTER_ANGLE_MOTOR,
+            followerAngleMotor = ToohardArmConstants.FOLLOWER_ANGLE_MOTOR,
+            masterElevatorMotor = ToohardArmConstants.MASTER_ELEVATOR_MOTOR,
+            followerElevatorMotor = ToohardArmConstants.FOLLOWER_ELEVATOR_MOTOR;
+    private final TalonSRX elevatorEncoder = ToohardArmConstants.ELEVATOR_ENCODER;
+    private final StatusSignal<Double> angleEncoderPositionSignal = ToohardArmConstants.ANGLE_ENCODER_POSITION_SIGNAL;
+    private final StatusSignal<Double> angleEncoderVelocitySignal = ToohardArmConstants.ANGLE_ENCODER_VELOCITY_SIGNAL;
+
+    @Override
+    protected void updateInputs(ArmInputsAutoLogged inputs) {
+        inputs.angleMotorVoltage = masterAngleMotor.getBusVoltage();
+        inputs.elevatorMotorVoltage = masterElevatorMotor.getBusVoltage();
+
+        inputs.elevatorPositionRevolutions = getElevatorPositionRevolutions();
+        inputs.elevatorVelocityRevolutionsPerSecond = getElevatorVelocityRevolutionsPerSecond();
+
+        inputs.angleEncoderPositionSignal = getAngleEncoderPositionSignal();
+        inputs.angleEncoderVelocitySignal = getAngleEncoderVelocitySignal();
+    }
+
+    private double getElevatorPositionRevolutions() {
+        return Conversions.magTicksToRevolutions(elevatorEncoder.getSelectedSensorPosition());
+    }
+
+    private double getElevatorVelocityRevolutionsPerSecond() {
+        return Conversions.perHundredMsToPerSecond(Conversions.magTicksToRevolutions(elevatorEncoder.getSelectedSensorVelocity()));
+    }
+
+    private double getAngleEncoderPositionSignal() {
+        return angleEncoderPositionSignal.getValue();
+    }
+
+    private double getAngleEncoderVelocitySignal() {
+        return angleEncoderVelocitySignal.getValue();
+    }
```

Comment:
1. You forgot to refresh 
2. What about units? 
3. This doesn't return a signal, this returns a value